release-notes/docs/3.5.0.md

```diff
@@ -0,0 +1,179 @@
+|---|---|---|---|
+| Rendering the History tab | 7247ms | 1843ms | 75% |
+| Starting the application | 1020ms | 875ms | 14% |
+| Idle memory usage | 453MB | 202MB | 55% |
+
+A further optimization was made to the History detail pane by improving how we link to Patients, which reduced rendering time from 1002ms to 732ms an improvement of 27%. [medic#5455](https://github.com/medic/medic/issues/5455)
+
+### Sentinel processing
+
+By reducing the the throttling of change processing and reducing the number of database writes we have greatly improved the rate at which Sentinel processes changes. Clearing a backlog of 200 documents used to take 67 seconds, but with 3.5.0 this has been reduced to 33 seconds, a 50% reduction. [medic#4128](https://github.com/medic/medic/issues/4128).
+
+To make Sentinel even more responsive to changes, transitions are now executed synchronously when messages are received. This means that replies are generated immediately and the user is be notified of the success or error of their message as soon as possible. [medic#4857](https://github.com/medic/medic/issues/4857).
+
+### Reducing the app size
+
+After some analysis it was found that the webapp included font files that weren't required. Certain files weren't minified, and some others weren't compressed. These fixes combined reduced the size by 2.6MB making the application easier to download and update. [medic#5260](https://github.com/medic/medic/issues/5260), [medic#5489](https://github.com/medic/medic/issues/5489), and [medic#5490](https://github.com/medic/medic/issues/5490).
+
+## Sort muted contacts
+
+Muted contacts are now sorted to the bottom of the contact list so you can more easily see all your active contacts. [medic#5596](https://github.com/medic/medic/issues/5596).
+
+![Sort muted contacts](images/3.5.0-sort-muted-contacts.png)
+
+## SIH integration
+
+Version 3.5.0 includes an integration with SIH, a facility based application in Mali. This enables patient referral information to be sent to a clinic and treatment data to be returned, closing the loop between the Community Health Worker and the clinic. [medic#5499](https://github.com/medic/medic/issues/5499), and [medic#4813](https://github.com/medic/medic/issues/4813).
+
+## And more...
+
+### Features
+
+- [medic#1850](https://github.com/medic/medic/issues/1850): Report version of application and forms being used
+- [medic#4811](https://github.com/medic/medic/issues/4811): Expose SMS sending capabilities in android wrapper
+- [medic#4812](https://github.com/medic/medic/issues/4812): Support XForms that "compile" into SMS messages
+
+### Improvements
+
+- [horticulturalist#32](https://github.com/medic/horticulturalist/issues/32): Horti should by default install release version and not master
+- [medic-conf#117](https://github.com/medic/medic-conf/issues/117): Bulk user import
+- [medic-conf#135](https://github.com/medic/medic-conf/issues/135): Create-user function always creates a new place
```

Comment:
Consider linking to https://github.com/medic/medic-conf/blob/master/release-notes.md#20

Reply:
I'll have to think on this some more. My immediate thought is that medic-conf is a separate product so I don't want to confuse readers by linking there. But that means the issues should be included here either.